### PR TITLE
fix(dropdown): change disabled styles for dropdown

### PR DESCRIPTION
**Pull request by ranst91-u**
Preview: https://static-staging.usabilla.com/stylabilla/pr/blup-407-dropdown-disabled/docs/section-components.html#kssref-components-dropdown-disabled

## What to test? Scope of the test?

Disabled dropdown styles changed
## Reviews

Reviewers: @chiel @MLKiiwy @thefrontendwizard @kagnamazian @realdreamer @DavideBruner @ranst91-u @galagmon @Maruceau @gsbkr @michelvdhoek 

Required: 1 FE approval - 1 UI approval

Answers design requests mentioned here:
https://github.com/usabilla/analysis-app/pull/162

### CODE REVIEW

**Comment by ranst91-u** [comment]
> Codewise looks fine to me, but won't removing the arrow leave the user wondering what this thing is?

Yes, we just decided to put it back.

**Comment by chiel** [comment]
Ah right, I see now. Maybe rebase these commits into one before merging. :)

File: src/components/_dropdown.scss [hunk 1]
```diff
@@ -58,8 +60,15 @@
     }
   }
 
-  .#{$ns}input:disabled ~ .#{$ns}icon {
-    fill: $sb-color-light-1;
+  .#{$ns}input:disabled {
+    &:hover {
+      box-shadow: none;
+      cursor: auto;
```
Comment (kagnamazian) [inline src/components/_dropdown.scss]: why not `not-allowed`?

Comment (ranst91-u) [reply src/components/_dropdown.scss]: @galagmon @Maruceau @gsbkr

**Comment by galagmon** [comment]
Shouldn't we have something like this for a disabled curser?
image

**Comment by ranst91-u** [comment]
@galagmon @kagnamazian changed to 'not allowed' cursor

**Pull request merged**